diff --git a/plugin/DoxygenToolkit.vim b/plugin/DoxygenToolkit.vim
index 37d51a6..45303b6 100644
--- a/plugin/DoxygenToolkit.vim
+++ b/plugin/DoxygenToolkit.vim
@@ -448,9 +448,9 @@ function! <SID>DoxygenLicenseFunc()
   mark d
   let l:date = strftime("%Y")
   exec "normal O".strpart( s:startCommentBlock, 0, 1 )
-  exec "normal A".strpart( s:startCommentBlock, 1 ).substitute( g:DoxygenToolkit_licenseTag, "\<enter>", "\<enter>".s:interCommentBlock, "g" )
+  exec "normal A".strpart( s:startCommentBlock, 1 ).substitute( g:DoxygenToolkit_licenseTag, "\<enter>", "\<enter>"." ".s:interCommentBlock, "g" )
   if( s:endCommentBlock != "" )
-    exec "normal o".s:endCommentBlock
+    exec "normal o"." ".s:endCommentBlock
   endif
   if( g:DoxygenToolkit_licenseTag == s:licenseTag )
     exec "normal %jA".l:date." - ".g:DoxygenToolkit_authorName
