diff --git a/plugin/autoload_cscope.vim b/plugin/autoload_cscope.vim
index 6e8d96d..79a32d9 100644
--- a/plugin/autoload_cscope.vim
+++ b/plugin/autoload_cscope.vim
@@ -161,7 +161,7 @@ function s:Cycle_csdb()
       if !cscope_connection(3, "out", b:csdbpath)
         let save_csvb = &csverb
         set nocsverb
-        exe "cs add " . b:csdbpath . "/cscope.out " . b:csdbpath
+        exe "cs add " . b:csdbpath . "/cscope.out " . b:csdbpath "-C"
         set csverb
         let &csverb = save_csvb
       endif
@@ -174,10 +174,12 @@ endfunc
 " auto toggle the menu
 augroup autoload_cscope
  au!
- au BufEnter *.[chly]  call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
- au BufEnter *.cc      call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
- au BufUnload *.[chly] call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
- au BufUnload *.cc     call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
+ " au BufEnter *.[chly]  call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
+ " au BufEnter *.cc      call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
+ " au BufUnload *.[chly] call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
+ " au BufUnload *.cc     call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
+ au BufEnter * call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
+ au BufUnload * call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
 augroup END
 
 let &cpo = s:save_cpo
