diff --git a/plugin/autoload_cscope.vim b/plugin/autoload_cscope.vim
index 6e8d96d..247e3d7 100644
--- a/plugin/autoload_cscope.vim
+++ b/plugin/autoload_cscope.vim
@@ -174,10 +174,12 @@ endfunc
 " auto toggle the menu
 augroup autoload_cscope
  au!
- au BufEnter *.[chly]  call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
- au BufEnter *.cc      call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
- au BufUnload *.[chly] call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
- au BufUnload *.cc     call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
+ " au BufEnter *.[chly]  call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
+ " au BufEnter *.cc      call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
+ " au BufUnload *.[chly] call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
+ " au BufUnload *.cc     call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
+ au BufEnter * call <SID>Cycle_csdb() | call <SID>Cycle_macros_menus()
+ au BufUnload * call <SID>Unload_csdb() | call <SID>Cycle_macros_menus()
 augroup END
 
 let &cpo = s:save_cpo
